lib/flip_card.dart

```diff
@@ -178,6 +178,21 @@ class FlipCardState extends State<FlipCard>
     });
   }
 
+  /// Flip the card without playing an animation
+  void toggleCardWithoutAnimation() {
```

Comment:
The issue you faced was probably because of the controller status being out of sync with `isFront`.

I'm not a fan of this duration hack. Can you make it work without it by setting the [value](https://api.flutter.dev/flutter/animation/AnimationController/value.html) directly to 0/1 maybe?